modules/osd-create-cluster-gcp-account.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="osd-create-cluster-gcp-account_{context}"]
+= Creating a cluster on GCP with Google Cloud Marketplace
+
+When creating an {product-title} (OSD) cluster on Google Cloud through the OpenShift Cluster Manager Hybrid Cloud Console, customers can select Google Cloud Marketplace as their preferred billing model. This billing model allows Red Hat customers to take advantage of their link:https://cloud.google.com/docs/cuds[Google Committed Use Discounts (CUD)] towards {product-title} purchased through the Google Cloud Marketplace. Additionally, OSD pricing is consumption-based and customers are billed directly through their Google Cloud account.
+
+.Procedure
+
+. Log in to {cluster-manager-url} and click *Create cluster*.
+
+. In the *Cloud* tab, click *Create cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. Under *Billing model*, configure the subscription type and infrastructure type:
+.. Select the *On-Demand* subscription  type.
+.. From the dropdown menu, select *Google Cloud Marketplace*.
+.. Select the *Customer Cloud Subscription* infrastructure type.
+.. Click *Next*.
+. On the *Cloud provider* page, read the provided prerequisites and the Google terms and conditions. Add your service account key.
+.. Click the *Review Google Terms and Agreements* link.
+.. If you agree to the Google terms and agreements and want to proceed, continue completing the steps in the {product-title} cluster create wizard.
```

Comment:
It is `If you agree...`. If the customer wants to continue, it must accept the terms and agreements.

And after accepting `the terms and agreements`, do we still need to select the checkbox to `acknowledge that you have read and completed all of the prerequisites`?

Reply:
Good question. I have a question out to the mock-up designer to see what option the user has to make (ie radio button, checkbox, etc) once they read those terms and agreements.

Reply:
Got response, will update docs.

Reply:
Yes, we still need the checkbox at the end of the page. There is no change to this part i.e. to the placement of this checkbox. We are also added a specific call to action for reviewing Google terms and agreements as shown in this design doc https://docs.google.com/document/d/1xpRCZ0QYWRuvVtCEycWrHwXWSIepzUkgb8Nobqf3Dlc/edit#heading=h.ve1e69gtx5pl